style_guides/accessibility_guide.md

```diff
@@ -0,0 +1,108 @@
+# Accessibility (A11Y) Guide
+
+This document provides some technical guidelines how to prevent several common
+accessibility issues.
+
+## Interactive elements
+
+### Use `<button>` and `<a>`
```

Comment:
Can we change the title of this to "Use `<button>` and `<a href>`? I think we should call out the importance of href as often as possible, because otherwise it will be very easy for people to think that just `<a>` by itself is sufficient.